src/containers/auth/forgotPassword/index.js

```diff
@@ -75,9 +75,9 @@ class ForgotPassword extends PureComponent {
     const valid = this.preValidateForm({ email });
     if (valid) {
       const response = await forgotPasswordRequest({ email });
-      const { status, data } = response;
-      const shownMessage = status < 300 ? 'successShow' : 'errorShow';
-      this.showCurrentMessage({ keys: [shownMessage], message: data });
+      const { status, message } = response;
+      const shownMessage = status < 300 ? ['successShow'] : ['formShow', 'validationShow'];
+      this.showCurrentMessage({ keys: shownMessage, message });
     }
```

Comment:
We already have a modal component, so wrap your component on it.

I don't like showMessage component and how you tied with statuses. As I understood, you use it only in your component, why you can't just use component state and display these messages depends on this state

Reply:
Done